src/source/Sctp/Sctp.c

```diff
@@ -205,64 +221,50 @@ STATUS sctpSessionWriteDcep(PSctpSession pSctpSession, UINT32 streamId, PCHAR pC
 {
      ENTERS();
      STATUS retStatus = STATUS_SUCCESS;
-     struct sctp_sendv_spa spa;
      PBYTE pPacket = NULL;
      UINT32 pPacketSize = SCTP_DCEP_HEADER_LENGTH + pChannelNameLen;
 
      CHK(pSctpSession != NULL && pChannelName != NULL, STATUS_NULL_ARG);
      CHK((pPacket = (PBYTE) MEMCALLOC(1, pPacketSize)) != NULL, STATUS_NOT_ENOUGH_MEMORY);
 
+     MEMSET(&pSctpSession->spa, 0x00, SIZEOF(struct sctp_sendv_spa));
+     MEMSET(pSctpSession->pPacket, 0x00, SIZEOF(pSctpSession->pPacket));
```

Comment:
What are you trying to achieve here? Normally, you don't memset a pointer. You can simply assign a NULL

Reply:
The naming is causing confusion. I intend to MEMSET the character array. Fixed naming to avoid the confusion